documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -12,54 +12,11 @@ IMPORTANT: If you are installing Raspberry Pi OS, and intend to run it headless,
 
 === Configuring Networking
 
-You will need to define a `wpa_supplicant.conf` file for your particular wireless network. Put this file onto the boot folder of the SD card. When the Raspberry Pi boots for the first time, it will copy that file into the correct location in the Linux root file system and use those settings to start up wireless networking.
+To configure network access at first boot in a headless Raspberry Pi, use the advanced settings menu in the Imager application. Enter a the SSID and password for your preferred wireless network. Your Raspberry Pi will use these credentials to connect to the network on first boot. Some wireless adapters and some Raspberry Pi boards do not support 5Ghz networks; check the documentation for your wireless module to ensure compatibility with your preferred network.
```

Comment:
'Imager' -> 'Raspberry Pi Imager' ?

"Enter a the SSID" doesn't sound quite right :wink: